### ISSUE TITLE
Set uwsgi logs properly

### ISSUE THREAD

**Issue by holandes22** [holandes22/rafee#35]
Currenly they are written to /var/log supervisor/uwsgi , maybe is better to redirect them to /var/log/uwsgi (the folder needs to be created)

**Comment by tzurE** [comment]
I've added to the file at provisioning/roles/supervisor/templates/supervisor/uwsgi.conf.j2
the lines - 
[program:uwsgi]
.......#like before
stderr_logfile=/var/log/uwsgi/stdError.log
stdout_logfile=/var/log/uwsgi/stdout.log

are these enough? not sure how to test it since uwsgi seems to be working fine.
I took the example from the celery conf file...

**Comment by holandes22** [comment]
yes basically. Not so sure about the camelCase named file. Please use a standard format

**Comment by tzurE** [comment]
oh, yeah. sorry. ill change it to lower case.
